src/core/hw/aes/key.cpp

```diff
@@ -153,6 +203,174 @@ void LoadBootromKeys() {
     }
 }
 
+void LoadNativeFirmKeysOld3DS() {
+    // Use the save mode native firm instead of the normal mode since there are only 2 version of it
+    // and thus we can use fixed offsets
+
+    constexpr u64_le save_mode_native_firm_id_low = 0x0004013800000003;
```

Comment:
```suggestion
    constexpr u64 save_mode_native_firm_id = 0x00040138'00000003;
```
You specified a full 64-bit ID here so there is no "_low" "_high" id. Also, as this doesn't go through any guest memory, it doesn't need to be `_le`